R/madrat.R

```diff
@@ -1,5 +1,5 @@
 .onLoad <- function(libname, pkgname) {
-  madrat::madratAttach(c(pkgname, "edgeTransport", "GDPuc", "mrcommons", "mrdrivers"))
+  madrat::madratAttach(c(pkgname, "madrat", "magclass", "edgeTransport", "GDPuc", "mrcommons", "mrdrivers"))
```

Comment:
`madrat` registers itself through `madratAttach()` → `getConfig()` → `initializeConfig()`.
`magclass` is not a madrat-based package, does not have calc/read/convert/tool functions and does not need to be registered.